inlang/source-code/sdk/src/loadProject.ts

```diff
@@ -154,9 +189,25 @@ export async function loadProject(args: {
 
 		// please don't use this as source of truth, use the query instead
 		// needed for granular linting
-		const [messages, setMessages] = createSignal<Message[]>()
+		// eslint-disable-next-line @typescript-eslint/no-unused-vars
+		const [messages, setMessages] = createSignal<Message[]>([])
+
+		const [loadMessagesViaPluginError, setLoadMessagesViaPluginError] = createSignal<
+			Error | undefined
+		>()
+
+		const [saveMessagesViaPluginError, setSaveMessagesViaPluginError] = createSignal<
+			Error | undefined
+		>()
+
+		const messagesQuery = createMessagesQuery(() => messages())
+
+		// const messageBaseFolderFolderPath = projectPath + "/messages"
+		// const messageFolderPath = messageBaseFolderFolderPath + "/v1"
+		const messageLockFilePath = projectPath + "/messagelock"
```

Comment:
We should add a .gitignore underneath the inlang folder and add the messagelock file to it

Reply:
How about renaming lockFilePath => lockDirPath
I'm not sure we need the gitignore since the directory is empty.

I added sentry logging to the tasklist, so when we find a leftover lock dir (unexpected) it would be nice to record that.

Reply:
hahaha nice one! you are right - the naming confused me as well. its an empty folder no need to git ignore - thats great

Reply:
>  folder no need to git ignore  

in case you need a gitignore, introducing a gitignore seems OK. paraglide js also uses a generated gitignore file.

Reply:
renamed lockFilePath to lockDirPath in the code.